### PR TITLE
Added getNewestPodcasts API

### PR DESCRIPTION
**Pull request by brian-doherty**

### CODE REVIEW

File: db/model.go [hunk 1]
```diff
@@ -371,6 +371,10 @@ func (pe *PodcastEpisode) SID() *specid.ID {
 	return &specid.ID{Type: specid.PodcastEpisode, Value: pe.ID}
 }
 
+func (pe *PodcastEpisode) PID() *specid.ID {
+	return &specid.ID{Type: specid.PodcastEpisode, Value: pe.PodcastID}
```
Comment (sentriz) [inline db/model.go:375]:
```suggestion
func (pe *PodcastEpisode) PodcastSID() *specid.ID {
	return &specid.ID{Type: specid.Podcast, Value: pe.PodcastID}
```

**Review by sentriz — COMMENTED** [review]
looks pretty good to me, but I think the specid should be Podcast?

**Comment by brian-doherty** [comment]
You are correct. My mistake!

**Comment by sentriz** [comment]
looks like we just need to update the calls to PID()


image

**Comment by brian-doherty** [comment]
OK sorted.

**Comment by sentriz** [comment]
lookin good to me, cheers brian :+1: 

image

**Comment by sentriz** [comment]
btw i just noticed this

image

has it always been like this? :grin: never noticed

**Comment by brian-doherty** [comment]
LOL yeah. UX is clearly not my jam because I just let stuff like that slide by me.

**Pull request merged**